service/src/main/java/com/codeforcommunity/processor/SiteProcessorImpl.java

```diff
@@ -133,6 +134,12 @@ public GetSiteResponse getSite(int siteId) {
       throw new ResourceDoesNotExistException(siteId, "site");
     }
 
+    String neighborhood =
+        db.selectFrom(NEIGHBORHOODS)
+            .where(NEIGHBORHOODS.ID.eq(sitesRecord.getNeighborhoodId()))
+            .fetchOne()
```

Comment:
This can't be null, right? Just want to double check

Reply:
Correct, this is a required field